src/window/dxgi/src/lib.rs

```diff
@@ -190,20 +190,27 @@ impl DeviceExt for Device {
     }
 }
 
+
 /// Update the internal dimensions of the main framebuffer targets. Generic version over the format.
-pub fn update_views<Cf, D>(window: &mut Window, factory: &mut Factory, device: &mut D, width: u16, height: u16)
-            -> Result<h::RenderTargetView<Resources, Cf>, f::TargetViewError>
-where Cf: format::RenderFormat, D: DeviceExt
+pub fn update_view<Cf, D>(
+    window: &mut Window, factory: &mut Factory, device: &mut D,
+    width: u16, height: u16,
+    main_color: &mut h::RenderTargetView<Resources, Cf>,
+) where
+    Cf: format::RenderFormat + format::TextureFormat,
+    D: DeviceExt,
 {
-    
+    use core::Factory;
+    // first, replace the main view pointer with a dummy
+    // the code assumes this is the last standing reference, beware!
+    *main_color = factory.create_render_target(1, 1).unwrap().2;
```

Comment:
I'm not sure what's the purpose of this dummy target ? Looking at #1122 makes me believe that it's meant to propagate it to the application resize handling, which is not possible at this crate level. You could drop the given handle and return a new one ?

Reply:
> You could drop the given handle and return a new one ?

dropping is fine, but then you'd have to do some magic to prevent a double drop when the new one needs to overwrite the old one.
I chose the simplest approach here - overwrite the handle with a dummy target temporarily, so that the old one is freed.